containers/roles/foreman/files/wait_on_postgres.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
A nice future improvement would be to have this in a common role so this code isn't duplicated.